### ISSUE TITLE
It should be possible to generate flowtypes from properties defined with dashes

### ISSUE THREAD

**Issue by SpearThruster** [yayoc/swagger-to-flowtype#26]
Having something like this in the yaml file will break the generation (will fail when trying to generate the x-enumNames property) :

```
definitions:

  Status:
    type: string
    title: "Status"
    enum: ["s1","s2","s3"]
    x-enumNames: ["PENDING", "READY", "DONE"]
```

**Comment by SpearThruster** [comment]
I will create a PR until end of week.

**Comment by yayoc** [comment]
I published v0.2.3 includes your change, Thanks! @SpearThruster
